app/src/main/java/com/yennguyen/yen/moviedb_27/screen/home/ItemGenreViewModel.java

```diff
@@ -19,6 +23,13 @@ public void setGenre(Genre genre){
     }
 
     public void onItemClick(View view) {
```

Comment:
sao viewmodel của c toàn làm công việc của view thế này. nếu muốn dùng databinding để bắt sự kiện click cho item trên recycler cậu có thể làm thế này. Tạo một listener chứa hàm xử lý onClick-> Implement listener này cho view-> trong hàm khở tạp nhận thêm một listener nữa-> trong xml của item khai báo một variable nhận vào một biến thuộc kiểu của Listener-> Trong adapter gán cái listener nhận từ view cho file xml. Đó là cách để bắt onclick trong xml